Updated release process for unowned bugs (#281)
Babylonpartners/ios-playbook: Cookbook/Technical-Documents/ReleaseProcess.md

5. General bug fixing for minor regressions and tasks, that are not related to significant changes made by squads for their own features. For more complex ones:
    1. Delegate the bug to the relevant squad/person. 
    2. If the bug is too complex to be fixed within the release window, please toggle off the feature and inform the Product Manager. 

>     6. Bugs that aren't owned by any squad should be fixed by the release engineers. The support engineer(s) can help if additional help is needed.

Reviewer: [nit] it's in the nested list as 3rd point, but displayed correctly 🙃 
```suggestion
    3. Bugs that aren't owned by any squad should be fixed by the release engineers. The support engineer(s) can help if additional help is needed.
```

>     6. Bugs that aren't owned by any squad should be fixed by the release engineers. The support engineer(s) can help if additional help is needed.

Author: ```suggestion
6. Bugs that aren't owned by any squad should be fixed by the release engineers. The support engineer(s) can help if additional help is needed.
```

>     Note: Where there is no relevant squad with ownership, the release enginneer should attempt to fix the bug themselves or with the help of the support engineer(s) if they need additional support.

Reviewer 2: IMO this should be another point in the responsibilities instead of a note.

```suggestion
    6. Bugs that aren't own by any squad should be fixed by the release engineers. The support engineer(s) can help if additional support is needed.
```
